ballerina/schemaClient.bal

```diff
@@ -0,0 +1,277 @@
+        } else if 'table is sql:Error {
+            return 'table;
+        } else {
+            sql:TableDefinition tableDef = {
+                name: tableName,
+                'type: <sql:TableType>'table["TABLE_TYPE"]
+            };
+
+            if !(include == sql:NO_COLUMNS) {
+                sql:ColumnDefinition[] columns = [];
+                stream<record {}, sql:Error?> colResults = self.dbClient->query(
+                    `SELECT COLUMN_NAME, DATA_TYPE, COLUMN_DEFAULT, IS_NULLABLE FROM information_schema.columns 
+                     WHERE (table_schema=${self.database} and table_name = ${tableName});`
+                );
+                do {
+                    check from record {} result in colResults
+                        do {
+                            sql:ColumnDefinition column = {
+                                name: <string>result["COLUMN_NAME"],
+                                'type: <string>result["DATA_TYPE"],
+                                defaultValue: result["COLUMN_DEFAULT"],
+                                nullable: (<string>result["IS_NULLABLE"]) == "YES" ? true : false
+                            };
+                            columns.push(column);
+                        };
+                } on fail error e {
+                    return error("Error - recieved sql data is of type SQL:Error", cause = e);
+                }
+                check colResults.close();
+
+                tableDef.columns = columns;
+
+                if include == sql:COLUMNS_WITH_CONSTRAINTS {
+                    map<sql:CheckConstraint[]> checkConstMap = {};
+
+                    stream<record {}, sql:Error?> checkResults = self.dbClient->query(
+                        `SELECT CONSTRAINT_NAME, CHECK_CLAUSE FROM information_schema.check_constraints 
+                        WHERE CONSTRAINT_SCHEMA=${self.database};`
+                    );
+                    do {
+                        check from record {} result in checkResults
+                            do {
+                                sql:CheckConstraint 'check = {
+                                    name: <string>result["CONSTRAINT_NAME"],
+                                    clause: <string>result["CHECK_CLAUSE"]
+                                };
+
+                                string colName = <string>result["COLUMN_NAME"];
+                                if checkConstMap[colName] is () {
+                                    checkConstMap[colName] = [];
+                                }
+                                checkConstMap.get(colName).push('check);
+                            };
+                    } on fail error e {
+                        return error("Error - recieved sql data is of type SQL:Error", cause = e);
+                    }
+                    check checkResults.close();
+
+                    _ = checkpanic from sql:ColumnDefinition col in <sql:ColumnDefinition[]>tableDef.columns
```

Comment:
why we use `checkpanic` here

Reply:
checkpanic is used due to check causing errors, also at the current section of code tableDef.columns has already been verified to contain columns therefore checkpanic is used.